docs/report/report.tex

```diff
@@ -189,174 +189,164 @@ \subsection{Deployment Cycle}
 \subsection{Probleme während der Durchführung}
 \colorbox{red}{TODO:}
 
+
+
+
 \section{Projekt Meilensteine}
+
 \subsection{Meilenstein 1}
-\colorbox{red}{TODO:}Kurze Einleitung, von wann bis wann ging
+Der erste Meilenstein begann am 29.10.2020 und ging bis zum 11.11.2020. Für den Anfang des Projektes war hier die Planung, Struktur und das Vorgehen des Projektes der Fokus.
 \subsubsection{Ziel}
-\colorbox{red}{TODO:}was haben wir uns vorgenommen, was war das ziel, was wollten wir schaffen?
+Zum Ende des Meilensteins sollte die Struktur und das Grundgerüst für das Projekt fertig gestellt sein.
+Das umfasste sowohl das Konzept der Software-Architektur, die genutzten Technologien (später Technologie-Stack), einer Anforderungsanalyse mit dem Kunden sowie die Befüllung des Backlogs mit den ersten Tickets.Auch musste sich das Team damit beschäftigen, wie es das Projekt durchführen wollte und welche Management- und Teamorganisations-Methodiken dafür angewendet werden sollten.
+Um ein schnellen und reibungslosen  Start für die Entwicklung zu gewährleisten, sollte das Repository entsprechend vorbereitet werden.
```

Comment:
```suggestion
Um ein schnellen und reibungslosen Start für die Entwicklung zu gewährleisten, sollte das Repository entsprechend vorbereitet werden.
```

Reply:
umgesetzt in https://github.com/project-leek/project-leek/pull/409/commits/db7fedecbe3f12f759a2c504c4b9f0e5a7061fee